runtime/src/main/resources/application.properties

```diff
@@ -18,6 +18,7 @@ spring.h2.console.settings.web-allow-others=true
 spring.jackson.deserialization.fail-on-unknown-properties=true
 spring.hateoas.use-hal-as-default-json-media-type=false
 terraform.workspace.directory=xpanse_deploy_ws
+openapi.download-generator-client-url=https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/6.5.0/openapi-generator-cli-6.5.0.jar
```

Comment:
Just some suggestion:
can we add a property named `openapi.generator-client-version`?
If the `openapi.download-generator-client-url` specified, this package will be used.
If the  `openapi.generator-client-version` specified, we should get the specified version.
else we should get the latest version.

Reply:
There is no method to get lastest version of the generator-client-cli.jar from  the maven repo. I can use openapi.generator-client-version replace the openapi.download-generator-client-url in the properties file and join the complete URL with version in the OpenApiUtils.class. Is it Ok？